src/components/infoCard/InfoCard.tsx

```diff
@@ -28,7 +28,7 @@ type Props = WebUrlProps & {
   openWebScreen: (link: string) => void;
 };
 
-const mergeWebUrls = ({ webUrls, contact, contacts }: WebUrlProps) => {
+export const mergeWebUrls = ({ webUrls, contact, contacts }: WebUrlProps) => {
```

Comment:
please move this in our helpers area, as this method is not used just for info cards anymore. the component should stay clean for its purpose.

Reply:
Yeah, you're right. You can find this fix in this commit.https://github.com/smart-village-solutions/smart-village-app-app/pull/916/commits/081c27876bf3cbd07a86a776fd5aff7f5aef0b28